sfepy/scripts/resview.py

```diff
@@ -546,19 +545,25 @@ def pv_plot(filenames, options, plotter=None, step=None, annotations=None,
         if opts.get('l', options.outline):  # outline
             plotter.add_mesh(pipe[-1].outline(), color='k')
 
-        scalar = field
-        scalar_label = scalar
         is_vector_field = ((field is not None)
```

Comment:
This can be moved into `if 'g' in opts and is_point_field` block.

Reply:
is_vector_field is used also in streamlines?